pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "xcodeproj"
-version = "1.0.2"
+version = "1.1.0"
```

Comment:
Since it's a new python version, this should be bumped to 2.0.0

Reply:
Thanks, you're right, I'll get this and other projects fixed.